docs/site/components/Navigation.tsx

```diff
@@ -169,6 +170,65 @@ export const GitHubButton = () => {
   );
 };
 
+const useGetReleasedVersions = () => {
+  const [releasedVersions, setReleasedVersions] = React.useState<
+    {
+      label: string;
+      path: string;
+    }[]
+  >([]);
+
+  React.useEffect(() => {
+    const send = async () => {
+      const response = await fetch(
+        'https://api.github.com/repos/player-ui/player-ui.github.io/contents/'
+      );
+
+      const data = await response.json();
+      const versions = data
+        .filter((d) => d.type === 'dir' && d.name.match(/^v\d$/))
```

Comment:
Shouldn't this regex match any name that is a valid semvar version? Or are we organizing the docs folders differently?

Reply:
The docs are published by major -- so `v0`, `v1`,  etc 
I should fix it to handle `vXX` though

Reply:
The release folder is determined from 
https://github.com/player-ui/player/blob/900e3c6c11fc45127cd6252da40476e44ef5b345/scripts/release.sh#L57

Reply:
Yeah, minor/patch versions were going to be my followup question Also not sure where we landed on publishing docs for release candidates and canaries.

Reply:
Anything merged to `main` (effectively an RC) goes under `next` -- we don't have anything for PR canaries yet